espanol/rezando/paquete/NOSE/NOSE.md

```diff
@@ -1,2 +1,6 @@
 Pareciera ser una nota para Ester, pero ¿Quién era Ester?, asi que Beatito
 inicio la busqueda de Ester, inicio buscandola en:
+
+[el prostíbulo del pueblo.](prostibulo/prostibulo.md)
+
+[el Ayuntamiento del pueblo.](ayuntamiento/ayuntamiento.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.